slides/k8s/kubenet.md

```diff
@@ -30,7 +32,7 @@
 
 - No new protocol
 
-- Pods cannot move from a node to another and keep their IP address
+- The network implementation can decide how to allocate addresses
 
 - IP addresses don't have to be "portable" from a node to another
```

Comment:
"from a node to another" would be better as "from one node to another". "from a node to another" sorta sounds like "from a node to another non-node thing".

Reply:
Oh thanks!